### PR TITLE
amend ruuvi, add bparsite

### PR DESCRIPTION
**Pull request by Mircho**
Adapt the ruuvi example to use same methods everywhere
Add bparasite example

### CODE REVIEW

File: ble-ruuvi.js [hunk 1]
```diff
@@ -1,118 +1,125 @@
...
+	mqtt_topic: 'ruuvi',
+	event_name: 'ruuvi.measurement',
 };
 
-// https://docs.ruuvi.com/communication/bluetooth-advertisements/data-format-5-rawv2
-let RuuviParser = {
+let RUUVI_MFD_ID_LE = 0x9904;
+let RUUVI_DATA_FMT = 5;
 
-    getInt16BE: function(bytes) {
```
[the remark this answers is not in the document]
Comment (Mircho) [reply ble-ruuvi.js:11]: Tabs replaced by spaces.

[the remark this answers is not in the document]
Comment (kzyapkov) [reply ble-ruuvi.js:11]: this continues to be one massive whitespace change, not reviewable as is.

Either handle #13 first or revert the whitespace reformatting to make your actual changes visible.

File: ble-bparasite.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
+//For b-parasite data format see
```
Comment (kzyapkov) [inline ble-bparasite.js:1]: This can be broken out in a separate PR.

File: ble-bparasite.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+      temperature: getUInt16BE(env.slice(4)) / 100,
+      humidity: 100 * getUInt16BE(env.slice(6)) / 0x10000,
+      moisture: 100 * getUInt16BE(env.slice(8)) / 0x10000
+    };
+    if(hasLux) {
+      result.lux = getUInt16BE(env.slice(16));
+    }
+    return result;
+  }
+};
```
Comment (kzyapkov) [inline ble-bparasite.js:42]: The code relies on fixed offsets of the BTHome measurement elements, but this is not guaranteed. It'd be better if this supported actual BTHome v1 parsing.
